src/Core/Migration/V6_4/Migration1665267882RenameCountryVat.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Shopware\Core\Migration\V6_4;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Migration\Traits\ImportTranslationsTrait;
+use Shopware\Core\Migration\Traits\Translations;
+
+class Migration1665267882RenameCountryVat extends MigrationStep
+{
+    use ImportTranslationsTrait;
+
+    public function getCreationTimestamp() : int
+    {
+        return 1665267882;
+    }
+
+    public function update(Connection $connection) : void
+    {
+        $results = $connection->fetchFirstColumn('SELECT id FROM country WHERE iso = :iso AND iso3 = :iso3', ['iso' => 'VA', 'iso3' => 'VAT']);
+
+        $countryId = $results[0];
```

Comment:
you rely on the fact that the row was not deleted in the DB, you can not make any assumptions about the current DB state in migrations.

Therefore check if the id exists and do an early return and please add a testcase for this case too

BTW you could use `$connection->fetchOne(...)` which would give you the id or false if it does not exist, this makes the code cleaner IMHO as you don't have to deal with the result as array, because you are only interested in the first result